docker/sui-node/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM rust:1.60.0 AS chef
```

Comment:
As this is missing an entrypoint (and some configuration copy stanzas), I'd advise a comment here indicating how useful this is. Dockerfiles in a repo tend to be interpreted as ready even if they are, we don't want to confuse folks.